bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -1100,14 +1091,14 @@ void ReplicaImp::onMessage<PrePrepareMsg>(PrePrepareMsg *msg) {
     const bool slowStarted = (msg->firstPath() == CommitPath::SLOW || seqNumInfo.slowPathStarted());
 
     bool time_is_ok = true;
-    if (config_.timeServiceEnabled && msg->numberOfRequests() > 0) {
-      if (!time_service_manager_->hasTimeRequest(*msg)) {
-        LOG_WARN(CNSUS, "PrePrepare will be ignored");
+    if (config_.timeServiceEnabled) {
+      if (msg->getTime() <= 0) {
+        LOG_WARN(CNSUS, "No timePoint in the prePrepare, PrePrepare will be ignored");
```

Comment:
nit: Can we also print the value of msg->getTime(), will be useful in debugging in case any issues.

Reply:
Thanks @ukg19 , I will add this in the next commit.